cli/jobs/pipelines/automl/cli-automl-text-classification-multilabel-paper-categorization-pipeline/text-classification-multilabel-paper-categorization-pipeline.yml

```diff
@@ -39,3 +39,9 @@ jobs:
       mltable_output: ${{parent.jobs.text_classification_multilabel_node.outputs.best_model}}
     command: ls ${{inputs.mltable_output}}
     environment: azureml:AzureML-sklearn-0.24-ubuntu18.04-py37-cpu:1
+  register_model_job:
+    type: command
+    component: file:./components/component_register_model.yaml
+    inputs:
+      model_input_path: ${{parent.jobs.text_classification_multilabel_node.outputs.best_model}}
+      model_base_name: paper_categorization_model
```

Comment:
(nit) add a new line at the end of the file

Reply:
Changes Done

Reply:
Changes Done